hrvisa/tests/phpunit/CRM/AllTests.php

```diff
@@ -1,68 +0,0 @@
-<?php
```

Comment:
- hrabsence and all its traces will be removed from 4.7 and will be kept on 4.5 branch only once you finish hrleaveandabsences , so no need to migrate it . 

- hrjob will be removed soon to so there is no need for that .

- for hrleaveandabsences I didn't touch anything related to it since you are working on it and also I think its up to date with latest test practices ... but I will delete these files